Chapters/4-7-Chapter29.md

```diff
@@ -0,0 +1,231 @@
+
+I watched them disappear into the night after Setre, looked back at the guardians' light ebb into the darkness. I felt history repeat itself in a different hues. It was the same pattern in Pomri's capture, in Fajro's plan, in Shinna's form. A pattern of people trying to see through the darkness, blindly taking everything they can - even other people. I felt the pain of all who were lost in it: like cuts that bled together. Like kanosin and blood.
+
+A hand fell upon my shoulder. "Quite the ruler, isn't she?" said Ramne.
+
+I turned around, tripping over a rock in the darkness. I looked for Laen and Boon, but they were long gone.
+
+"I must admit: when I was the leader of her faction, I didn't think much of Setre. But when Casra ran off with you to Rudjan, all the rage she hid inside of her came out. Must be something in that Holragi blood."
+
+"Go away," I said, shrugging him off and hiding my fear. "I don't have time."
+
+I headed back toward the shore, hoping to pull him away from Annu and toward the Guardians.
+
+"Now now," he said raising a dagger to my neck, "You're still a kaln. You've got plenty of time."
+
+I stopped abruptly to avoid the blade.
+
+He explained to me, "We've got a lot of catching up to do. So many things you missed while you were out."
+
+_Out? Was he there?_
+
+"Your nather is a tough cookie, you know? I figured she'd die in the fire or at least the chaos after. But somehow she survived; convinced people to let her rule. Even got her own special flow tattoo to prove it. Casra would be prou–"
+
+"Don't talk about my mother," I said. The blade of the dagger cleaving my breath in two.
+
+"Oh you think I didn't want hae to stay alive? Addict slaves are great. They never fight, they never speak. Sometimes they die but those monstrosities always come back. But... willing slaves? Oh they're even better. They die when you kill them, and they work so hard. All for the glory of their Ulukoma and its great leader Casra. They'd never have known we were selling the fruits of their labor for a premium price."
+
+"Shut up!"
+
+He gripped my arm and stooped down to look closely at me.
+
+"Casra could have made so much money for the two of you. But thi didn't have the willpower. Let's just say I added some fuel to a small fire named Shinna and things kind of spiralled from there."
+
+"Fuck you!"
+
+He looked at me through narrowed eyes.
+
+"You know, it sounds like Casra never taught you any manners. I'll be the man you always wanted me to be today, how about that?"
+
+I was furious, but I wouldn't him know that. Anger only made him happy.
+
+"After you, father."
+
```

Comment:
overall reaction to this chapter was meh. it shouldn't be the emotion your penultimate chapter drives out from the reader.